app/lib/features/chat/widgets/image_message_builder.dart

```diff
@@ -131,9 +130,9 @@ class ImageMessageBuilder extends ConsumerWidget {
             ? BorderRadius.circular(6)
             : BorderRadius.circular(15),
         child: ConstrainedBox(
-          constraints: BoxConstraints(
-            maxHeight: isReplyContent ? size.height * 0.2 : size.height * 0.3,
```

Comment:
This would probably create some confusion for end-user as image UI should appear shrinked as a original message of reply to make replied content more prominent as we see in common chat messengers...

Reply:
Good catch. Let me do the needful here.